features/testbot/via.feature

```diff
@@ -52,6 +52,31 @@ Feature: Via points
             | a,c,f     | ab,bcd,bcd,de,efg        |
             | a,c,f,h   | ab,bcd,bcd,de,efg,efg,gh |
 
+    Scenario: Alternative via points
+        Given the node map
+            |   | b | c | d |   |   |
+            | a |   |   |   |   | z |
+            |   | g | h | i | j |   |
+
+        And the ways
+            | nodes |
+            | ab    |
+            | bc    |
+            | cd    |
+            | dz    |
+            | ag    |
+            | gh    |
+            | hi    |
+            | ij    |
+            | jz    |
+
+        And the query options
+            | alt | true |
+
+        When I route I should get
+            | waypoints | route       | alternative    |
+            | a,z       | ab,bc,cd,dz | ag,gh,hi,ij,jz |
```

Comment:
since you have no via points, i would use 'from' and 'to' columns instead of 'waypoints'. it's easier to read and is used everywhere else when there are no via points. note that this scenario file is meant for testing via points specifically.